lib/src/components/NxToast/NxToastContainer.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useRef} from 'react';
+import FocusContext from './contexts';
+
+import { NxToastContainerProps, nxToastContainerPropTypes } from './types';
+
+export { Props } from './types';
+
+const NxToastContainer = (props: NxToastContainerProps) => {
+  const { children } = props;
+
+  const ref = useRef<HTMLDivElement | null>(null),
+      prevSelectedRef = useRef<Element | null>(null);
+
+  const body = document.getElementsByClassName('nx-body')[0];
+
+  const adjustFocus = () => {
+    //When closing Toasts with a mouse click, document.activeElement becomes nx-body, so need
+    //to specify not to assign selectedRef to body so the focus returns to the appropriate element
+    if (!ref.current?.contains(document.activeElement) && document.activeElement !== body) {
+      prevSelectedRef.current = document.activeElement;
+    }
+
+    if (ref.current) {
+      const closeBtns = ref.current.querySelectorAll('.nx-toast .nx-btn--close');
```

Comment:
`querySelectorAll` takes a type parameter. If you specify it you don't have to cast the result later
```suggestion
      const closeBtns = ref.current.querySelectorAll<HTMLButtonElement>('.nx-toast .nx-btn--close');
```